content/post/council-meeting-2022-07-18/chart.csv

```diff
@@ -1,26 +1,25 @@
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5716033&GUID=C591C535-E66E-45B9-9C10-3B8BACB92506&Options=&Search=,resolution pass,DC-1,Resolution Directing the City Administrator to Expand Compliance Evaluations and to Continue to Provide Annual Reports Regarding Contractor Compliance with Prevailing Wage Requirements,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,TRUE,absent,FALSE,TRUE,FALSE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5716033&GUID=C591C535-E66E-45B9-9C10-3B8BACB92506&Options=&Search=,resolution amendment fail,DC-1 Postponement,Postponing DC-1 until August 4th,FALSE,FALSE,TRUE,FALSE,TRUE,FALSE,FALSE,absent,TRUE,FALSE,TRUE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution pass,DC-2,"Resolution to Organize a Joint Meeting of Interested Individuals from the Environmental Commission, Transportation Commission, Energy Commission, A2ZERO Ambassadors, and other Community Stakeholders to Identify Opportunities for Greater Coordination around Transportation, Pedestrian Safety, and Sustainability Initiatives Related to Sidewalk Expansion and Maintenance",TRUE,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,TRUE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment pass,DC-2 Amendment,Add commission on disabilities to the list of groups involved in the special meeting,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,TRUE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment fail,DC-2 Amendment,Add Hickory Lane (Racket Club) sidewalk project to the list of stuff for the group to work on.,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,recused,absent,TRUE,FALSE,FALSE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment pass,DC-2 Recusal,Recusal for CM Grand due to racket club interest in Hickory Lane,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,FALSE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment pass,DC-2 Amendment,Remove final resolved clause for the city administrator to evaluate members of commissions.,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5706387&GUID=C52C4AD3-67F8-40B5-8C0B-4804FA72BB37&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,nomination pass,MC-1,Appointment - Confirmations,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE
```

Comment:
counts eyer as present for voice votes, even though she was absent from the meeting

Reply:
This is a limitation of the underlying Legistar data. Some agenda items with voice votes have individual vote info and show anybody who was absent, just all [present] members' votes are "null" (this is from CA-12)

```
      "EventItemVoteInfo": [
        {
          "VoteId": 72071,
          "VoteGuid": "15D86E0E-22A0-4614-A70B-EA5FC7A320EA",
          "VoteLastModifiedUtc": "2022-07-18T23:49:07.423",
          "VoteRowVersion": "AAAAAAFXu7g=",
          "VotePersonId": 749,
          "VotePersonName": "Christopher Taylor",
          "VoteValueId": null,
          "VoteValueName": null,
          "VoteSort": 1467,
          "VoteResult": null,
          "VoteEventItemId": 170408
        },
        {
          "VoteId": 72072,
          "VoteGuid": "4B932622-F459-4B26-B759-006527E0FB23",
          "VoteLastModifiedUtc": "2022-07-18T23:49:07.43",
          "VoteRowVersion": "AAAAAAFXu7c=",
          "VotePersonId": 718,
          "VotePersonName": "Julie Grand",
          "VoteValueId": null,
          "VoteValueName": null,
          "VoteSort": 1468,
          "VoteResult": null,
          "VoteEventItemId": 170408
        },
...
        {
          "VoteId": 72080,
          "VoteGuid": "8127F5D5-E3AD-4350-A6D8-5791820E7DCB",
          "VoteLastModifiedUtc": "2022-07-18T23:49:07.44",
          "VoteRowVersion": "AAAAAAFXu78=",
          "VotePersonId": 1626,
          "VotePersonName": "Jen Eyer",
          "VoteValueId": 19,
          "VoteValueName": "Absent",
          "VoteSort": 2047,
          "VoteResult": 0,
          "VoteEventItemId": 170408
        },
...
      ]
```

Other agenda items with voice votes show ... nothing (this is from MC-1):

```
      "EventItemVoteInfo": []
```

There's no perfect answer to this. A CM could be absent for only part of the meeting, for example. So if the record doesn't show whether somebody was absent or not, we cannot with 100% accuracy infer that automatically.

Generally speaking, my position on voice votes is this: if we're going to generate a chart automatically without human intervention, then we **should not even attempt** to show how individual CMs voted. That's how the twitter bot approaches this. If you're comfortable with a requirement to have a human watch / review the video footage to fill these in, then okay. But this particular issue is probably a "WONTFIX" situation :)